test/test_wiki_trac_rst_convert.py

```diff
@@ -3,10 +3,10 @@
 from wiki_trac_rst_convert import convert_content
 
 
-class TracRstToVanillaRst(unittest.TestCase):
+class TracToGithubRst(unittest.TestCase):
```

Comment:
naming is complicated  ... but I think that the correct speling is GitHub ...also, when we have an abreviation like RST or URL, we keep upper case in the Python name.

so we prefer `getURL`  and not `getUrl` 

```suggestion
class TracToGitHubRST(unittest.TestCase):
```